src/Kripke/Core/Field.h

```diff
@@ -245,7 +245,7 @@ namespace Core {
         LType layout = RAJA::make_stride_one<LInfo::stride_one_dim>(m_chunk_to_layout[chunk_id]);
 
 #if (defined(KRIPKE_USE_HIP) || defined(KRIPKE_USE_CUDA)) && defined(KRIPKE_USE_CHAI)
-        return ViewType<Order, ElementType, ElementPtr, IDX_TYPES...>(Parent::m_chunk_to_data[chunk_id].getPointer(chai::GPU), layout);
+        return ViewType<Order, ElementType, ElementPtr, IDX_TYPES...>(Parent::m_chunk_to_data[chunk_id], layout);
```

Comment:
I would have replaced `getPointer(chai::GPU)` with `data(chai::GPU)`.

Reply:
Or is `Parent::m_chunk_to_data[chunk_id]` a raw pointer?

Reply:
That change was possible after changing the `ElementPtr` type to `ElementType *`. I'm not sure why this was working with previous compilers . . .